src/main/java/com/google/cloud/tools/app/impl/cloudsdk/CloudSdkAppEngineVersions.java

```diff
@@ -82,6 +83,7 @@ public void stop(VersionsSelectionConfiguration configuration) throws AppEngineE
     arguments.add("stop");
     arguments.addAll(configuration.getVersions());
     arguments.addAll(GcloudArgs.get("service", configuration.getService()));
+    arguments.addAll(GcloudArgs.get(configuration));
```

Comment:
What about `delete` and `list`?
I would actually factor out these 3 lines into a private method for reuse:
```Java
private addVersionsArguments(AppEngineVersions configuration, List<String> arguments) {
  arguments.addAll(configuration.getVersions());
  arguments.addAll(GcloudArgs.get("service", configuration.getService()));
  arguments.addAll(GcloudArgs.get(configuration));
}
```